runtime/memory.x

```diff
@@ -2,7 +2,7 @@
 
 ROM_ORG   = 0x00000000;
 META_ORG  = ROM_SIZE;
-ICCM_ORG  = 0x40002000;
+ICCM_ORG  = 0x40004000;
```

Comment:
Add comments to explain the iccm partition.

Reply:
Added.